README.md

```diff
@@ -18,3 +18,17 @@ GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
     video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
     xvimagesink d.audio ! queue ! audioconvert ! pulsesink
 ```
+
+Record website video + audio (with audiomixer)
```

Comment:
can you also add an audiomixer in the playback pipeline? :)

Reply:
I'm not sure I get what you mean. 
In this one?
```
GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
    cefsrc url="https://soundcloud.com/platform/sama" ! \
    video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
    xvimagesink d.audio ! queue ! audioconvert ! pulsesink
```

Reply:
yes

Reply:
Hmm interesting. I can't get it to work :/

```
GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
    cefsrc url="https://soundcloud.com/platform/sama" ! \
    video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
    xvimagesink audiotestsrc do-timestamp=true is-live=true  volume=0.01 ! audiomixer name=mix ! \
    queue ! audioconvert ! pulsesink \
    d.audio ! mix.
```
I can hear `audiotestsrc`, but can't `d.audio`

Reply:
hrm that's weird indeed, resolving this then but please open an issue, something's fishy :(

Reply:
actually, also works fine with larger queues:

```
gst-launch-1.0     cefsrc url="https://soundcloud.com/platform/sama" !     video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! videoconvert !     xvimagesink d.audio ! queue ! audioconvert ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! audiomixer name=mix ! autoaudiosink audiotestsrc is-live=true volume=0.0 ! queue ! mix.
```

Reply:
Ou I see I had to increase both queues. I tried before by just increasing one of them. Interesting.
Updated readme.md

Reply:
Actually. Do we increase queues? I checked `gst-inspect-1.0 queue` and 
```
  max-size-buffers    : Max. number of buffers in the queue (0=disable)
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Unsigned Integer. Range: 0 - 4294967295 Default: 200 
  max-size-bytes      : Max. amount of data in the queue (bytes, 0=disable)
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Unsigned Integer. Range: 0 - 4294967295 Default: 10485760 
  max-size-time       : Max. amount of data in the queue (in ns, 0=disable)
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Unsigned Integer64. Range: 0 - 18446744073709551615 Default: 1000000000 
```
From what I see we decrease them :D

Reply:
no we increase them to 3 seconds or an unlimited number of bytes / buffers

Reply:
Ahh I see was confused with `(0=disable)` thought it removes buffers. `-1` is wording what I'm used to